doctrine/doctrine-migrations-bundle/3.0/config/packages/doctrine_migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    migrations_paths:
+        # namespace is arbitrary but should be different from App\Migrations
+        # as migrations classes should NOT be autoloaded
+        'App\DoctrineMigrations': '%kernel.project_dir%/Migrations'
```

Comment:
Shouldn't we keep the same namespace as before?
https://github.com/symfony/recipes/blob/24cc9f60acdb5fd2331b588152acc6c6b2a18cc6/doctrine/doctrine-migrations-bundle/1.2/config/packages/doctrine_migrations.yaml#L5

Reply:
There was this https://github.com/doctrine/DoctrineMigrationsBundle/pull/278#discussion_r331081231 conversation suggesting to change it to more recent practices as using the app name space

Reply:
It in this case they should not be under App\ neither App\Migrations (comment) nor App\DoctrineMigr.... 

TLDR:
Should the comment talk about App\DoctrineMigrat... too? 🧐

Reply:
I do not see any bc breaks with it since recipes are mainly used for new installations. Do you have a particular reason to keep the old namespace?

Reply:
ive kept the same ns